client/src/screen/LoginPage/LoginPage.js

```diff
@@ -32,11 +33,8 @@ export default {
     methods: {
         onShow() {
             // if type = text -> convert type = password and vice_versa
-            if (this.inputInfos[1].type == "text") {
-                this.inputInfos[1].type = "password";
-            } else {
-                this.inputInfos[1].type = "text";
-            }
+            console.log(this.passwordIndex);
+            this.inputInfos[this.passwordIndex].type === "text" ? this.inputInfos[this.passwordIndex].type = "password" : this.inputInfos[this.passwordIndex].type = "text";
```

Comment:
- Toán tử 3 ngôi vẫn còn dài dòng, chưa chuẩn
- Nên viết
this.inputInfos[Constant.PASSWORD_INDEX].type = this.inputInfos[Constant.PASSWORD_INDEX].type === "text" ? "password" : "text"

Reply:
done